tests/fixtures/single_run/models/water_supply.py

```diff
@@ -31,14 +32,11 @@ def simulate(self, decisions, state, data):
         """
 
         # unpack inputs
-        raininess = data['raininess']['UK']['year']['value']
```

Comment:
These changes could cause an issue upstream when we merge in #46...  We'll deal with it then!